src/sentry/apidocs/hooks.py

```diff
@@ -149,6 +151,35 @@ class CustomGenerator(SchemaGenerator):
     endpoint_inspector_cls = CustomEndpointEnumerator
 
 
+def find_method_and_check_paginate(content, class_name) -> bool:
+    with open(content) as file:
+        file_content = file.read()
+
+    # Pattern to find the class body, either the class body is followed by another class or it's the end of the file
+    class_pattern = (
+        rf"(class {class_name}.*:(.*?)(?=^class\s.*))|(class {class_name}.*:\s(.*?)(?=\Z))"
+    )
+    class_match = re.search(class_pattern, file_content, re.DOTALL | re.MULTILINE)
+
+    if not class_match:
+        return False
+
+    class_body = class_match.group(0)
+
+    # Pattern to find the 'get' method within the class body, up to the next method or the end of the class
+    get_method_pattern = r"(def get.*:(.*?)(?=def\s))|(def get.*:(.*?)(?=\Z))"
```

Comment:
i think you can handle the replays case if you look for the next function definition that is either get, put, post, patch, or delete